app/ViewModels/Concerns/Transaction/HasPayload.php

```diff
@@ -107,7 +108,7 @@ public function multiPaymentRecipients(): ?array
             if (isset($amounts[$index])) {
                 $recipients[] = [
                     'address' => $address,
-                    'amount'  => $amounts[$index],
+                    'amount'  => UnitConverter::formatUnits($amounts[$index], 'ark'),
```

Comment:
I think this is supposed to be `wei` :thinking: it took me a bit to figure out what the issue was. The amount shown against each recipient does't match the total sent/received. If you look at the total balance of the recipients in the cards, there balance is zero (I presume due to rounding - that's another issue). I noticed that locally matched the demo site when looking at the example transaction in the card

There is a separate issue whereby multipayment transactions aren't shown in the recipient's transaction list